### PR TITLE
chore: update response models to match new specs (SDKCF-4017)

### PR DESCRIPTION
**Pull request by imaginaris**
# Description
* Fixed optionals in response models.
* Updates JSON stubs
* Removed `lowerBodyMessage`
* increased timeout in UI tests for slower machines

## Links
SDKCF-4017

# Checklist
- [X] I have read the [contributing guidelines](../CONTRIBUTING.md).
- [X] I wrote/updated tests for new/changed code
- [X] I removed all sensitive data **before every commit**, including API endpoints and keys
- [X] I ran `fastlane ci` without errors

### CODE REVIEW

File: RInAppMessaging/Assets/FullView.xib [hunk 1]
```diff
@@ -1,9 +1,9 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<document type="com.apple.InterfaceBuilder3.CocoaTouch.XIB" version="3.0" toolsVersion="18122" targetRuntime="iOS.CocoaTouch" propertyAccessControl="none" useAutolayout="YES" useTraitCollections="YES" useSafeAreas="YES" colorMatched="YES">
+<document type="com.apple.InterfaceBuilder3.CocoaTouch.XIB" version="3.0" toolsVersion="19162" targetRuntime="iOS.CocoaTouch" propertyAccessControl="none" useAutolayout="YES" useTraitCollections="YES" useSafeAreas="YES" colorMatched="YES">
```
Comment (donnie-jp) [inline RInAppMessaging/Assets/FullView.xib:2]: How are these changes related?
The removal of `messageLowerBody`?

File: RInAppMessaging/Classes/Models/Responses/PingResponse.swift [hunk 2]
```diff
@@ -115,14 +114,13 @@ internal struct MessagePayload: Codable {
 }
 
 internal struct Resource: Codable {
-    let assetsUrl: String?
```
Comment (donnie-jp) [inline RInAppMessaging/Classes/Models/Responses/PingResponse.swift:118]: Was this removed because the spec says it is "always NULL"?

**Review by donnie-jp — APPROVED** [review]

File: RInAppMessaging/Classes/Models/Responses/PingResponse.swift (hunk 2 above)
Comment (imaginaris) [reply RInAppMessaging/Classes/Models/Responses/PingResponse.swift:118]: Yes. This field was used anyway

File: RInAppMessaging/Assets/FullView.xib (hunk 1 above)
Comment (imaginaris) [reply RInAppMessaging/Assets/FullView.xib:2]: Yes
